app/javascript/elm/src/Main.elm

```diff
@@ -471,7 +484,7 @@ viewMobileFilters model =
     form [ Attr.class "filters-form" ]
         [ viewParameterFilter model.selectedParameter
         , Popup.viewPopup TogglePopupState SelectParameter model.isPopupExtended model.popup
-        , viewLocation model.location
+        , viewLocation model
```

Comment:
What do you think about passing `viewLocation model.location model.indoor`? It has a benefit of showing exactly which parts of the model are used in the function.

Reply:
I like it